spyrograph/_trochoid.py

```diff
@@ -547,6 +547,8 @@ def _validate_theta(
                 theta_step = .1
             thetas = np.arange(theta_start, theta_stop, theta_step)
         thetas = np.array(thetas)
+        if multiple_thetas:
+            raise ValueError("An empty list of thetas was passed in as argument.")
```

Comment:
So close! This is exactly where we needed to put this so really good job identifying the right location! 👍 

The only thing we have to change is instead of checking `multiple_thetas` let's change this line to 

```python
if len(thetas) == 0:
    raise ValueError("An empty list of thetas was passed in as argument.")
```

Notice that I changed `if multiple_thetas:` to `if len(thetas) == 0`. This changes the behavior so that if the length of the `thetas` list is empty at this point _then_ we raise the error